CRM/Contribute/Form/Task/Invoice.php

```diff
@@ -298,13 +298,7 @@ public static function printPDF($contribIDs, &$params, $contactIds) {
       $invoiceDate = date("F j, Y");
       $dueDate = date('F j, Y', strtotime($contributionReceiveDate . "+" . $prefixValue['due_date'] . "" . $prefixValue['due_date_period']));
 
-      if ($input['component'] == 'contribute') {
-        $lineItem = CRM_Price_BAO_LineItem::getLineItemsByContributionID($contribID);
-      }
-      else {
-        $eid = $contribution->_relatedObjects['participant']->id;
-        $lineItem = CRM_Price_BAO_LineItem::getLineItems($eid, 'participant', NULL, TRUE, FALSE, TRUE);
-      }
+      $lineItem = CRM_Price_BAO_LineItem::getLineItemsByContributionID($contribID);
```

Comment:
Can you confirm if your line doesn't cause regression for CRM-16250 ? Since I guess  the line `CRM_Price_BAO_LineItem::getLineItems($eid, 'participant', NULL, TRUE, FALSE, TRUE);` was written to handle multiple participant(shouldn't be a problem since same contribution id is shared between participant and additional participant)

https://github.com/civicrm/civicrm-core/blob/master/CRM/Price/BAO/LineItem.php#L237

Reply:
That's correct, the line items are created with the same `contribution_id`, like this:
``` json
// civicrm_line_item table export
{
	"civicrm_line_item":
	[
		{
			"id": 238,
			"entity_table": "civicrm_participant",
			"entity_id": 226,
			"contribution_id": 91,
			"price_field_id": 47,
			"label": "Option 1",
			"qty": 1,
			"unit_price": 10,
			"line_total": 10,
			"participant_count": 1,
			"price_field_value_id": 77,
			"financial_type_id": 4,
			"non_deductible_amount": 0,
			"tax_amount": 2
		},
		{
			"id": 239,
			"entity_table": "civicrm_participant",
			"entity_id": 227,
			"contribution_id": 91,
			"price_field_id": 47,
			"label": "Option 2",
			"qty": 1,
			"unit_price": 10,
			"line_total": 10,
			"participant_count": 1,
			"price_field_value_id": 78,
			"financial_type_id": 4,
			"non_deductible_amount": 0,
			"tax_amount": 2
		}
	]
}
```

RE CRM-16250, here's a generated invoice for same Registration (Contact 1 + Contact 2) same PriceSet and fee levels, but registering the participants through Event page (CiviEvent) instead of API, with this patch applied.
![contribution_through_event_page](https://user-images.githubusercontent.com/3038096/51288688-4497ed00-19f5-11e9-97d2-c52323eda9a3.png)

And here it's export as well:
``` json
{
	"civicrm_line_item":
	[
		{
			"id": 240,
			"entity_table": "civicrm_participant",
			"entity_id": 228,
			"contribution_id": 92,
			"price_field_id": 47,
			"label": "Option 1",
			"qty": 1,
			"unit_price": 10,
			"line_total": 10,
			"participant_count": 1,
			"price_field_value_id": 77,
			"financial_type_id": 4,
			"non_deductible_amount": 0,
			"tax_amount": 2
		},
		{
			"id": 241,
			"entity_table": "civicrm_participant",
			"entity_id": 229,
			"contribution_id": 92,
			"price_field_id": 47,
			"label": "Option 2",
			"qty": 1,
			"unit_price": 10,
			"line_total": 10,
			"participant_count": 1,
			"price_field_value_id": 78,
			"financial_type_id": 4,
			"non_deductible_amount": 0,
			"tax_amount": 2
		}
	]
}
```